### ISSUE TITLE
Can't log in to my account / Everything about Mojang is shown down in MultiMC

### ISSUE THREAD

**Issue by ByJumperX4** [MultiMC/MultiMC5#3432]
System Information
-----------------------------
MultiMC version: 0.6.11-custom ( I made no manual changes, it just says that)

Operating System: Bedrock Linux w/ Gentoo GNU/Linux Hardened ( Runit/OpenRC + SELinux) as main stratum ( x86_64 architecture )

Summary of the issue or suggestion:
----------------------------------------------

Everything was working fine. But since I've converted my Gentoo install into a Gentoo Hardened with SELinux install, MultiMC says that mojang is down (now for 3 days). Everything else is fine on my system.

What should happen: 
------------------------------

Get working mojang status so I can log in to my minecraft account

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------

1. Convert a gentoo install to a gentoo hardened + selinux install

2. Launch multimc

3. See that you can't play with a minecraft account and you're only in offline mode

Suspected cause:
---------------------------

Some SSL/TLS problem

Logs/Screenshots:
----------------------------

Logs available here: https://paste.ramle.be/hidocatoni.py

![image](https://user-images.githubusercontent.com/48773926/98845694-5fe7ae00-244e-11eb-9453-bc5a57bc21cd.png)

**Comment by kb-1000** [comment]
How did you install MultiMC?

**Comment by ByJumperX4** [comment]
with my package manager (portage)

**Comment by kb-1000** [comment]
From where?

**Comment by ByJumperX4** [comment]
from the jorgicio overlay ? but it doesn't change anything as it was working before

**Comment by Zetabite** [comment]
well depends. If they just use the existing approved repos directly, thats fine, but i doubt that

**Comment by QuImUfu** [comment]
You could look into https://wiki.gentoo.org/wiki/SELinux/Logging and look if SELinux is causing that issue. If it is, fix it and maybe document it in the MultiMC wiki.

**Comment by kb-1000** [comment]
That would have to be fixed in the distro package, most likely by adding a policy module.
I use MultiMC on SELinux too (on Fedora) and it works just fine.

**Comment by ByJumperX4** [comment]
SELinux doesn't seem to block anything when I launch MultiMC

**Comment by QuImUfu** [comment]
do you have https://wiki.gentoo.org/wiki/Certificates#OpenSSL-compatible_ca-certificates installed?  
`The issuer certificate of a locally looked up certificate could not be found` lines in your log make me think that either your system is lacking default certificates or MultiMC can not access them for some reason.

**Comment by ByJumperX4** [comment]
```
$ ~> emerge --info | grep ca-certificate
CONFIG_PROTECT_MASK="/etc/ca-certificates.conf /etc/dconf /etc/env.d /etc/fonts/fonts.conf /etc/gconf /etc/gentoo-release /etc/revdep-rebuild /etc/sandbox.d /etc/terminfo"
$ ~> eix -I ca-certif
[I] app-misc/ca-certificates
     Available versions:  20200601.3.53 ~20200601.3.59 {cacert}
     Installed versions:  20200601.3.53(18:48:40 09/11/20)(-cacert)
     Homepage:            https://packages.debian.org/sid/ca-certificates
     Description:         Common CA Certificates PEM files

$ ~>
```

**Comment by yemouu** [comment]
I had this issue with LibreSSL 3.2.2 on Gentoo. The issue doesn't appear on LibreSSL 3.2.1.

**Comment by ByJumperX4** [comment]
Thanks this solved ! I hope 3.2.3 won't be buggy
